### ISSUE TITLE
Missing link for Advanced Project Template in Guide

### ISSUE THREAD

**Issue by mtangoo** [yiisoft/yii2-apidoc#9]
To experience the problem
1. Open docs
2. Click Guide
3. Click Special Topics
4. Click Advanced Project Template

**Comment by cebe** [comment]
there is a link to the readme file on github:

```
* [Advanced Project Template](https://github.com/yiisoft/yii2-app-advanced/blob/master/docs/guide/README.md)
```
https://github.com/yiisoft/yii2/blob/master/docs/guide/README.md

website docs are currently not up to date, will fix that when I find time for it.

**Comment by mtangoo** [comment]
FYI I did build them using script from yiisoft/yii2/issues/3267

**Comment by cebe** [comment]
Now I see the problem, need to adjust the guide to behave well with external links...

**Comment by mtangoo** [comment]
Thats great. I hope to get time that I help with docs.
I see few TBD on  guide!

**Comment by CedricYii** [comment]
Already fixed by #75
